neurovault/apps/main/static/css/style.css

```diff
@@ -211,3 +211,17 @@ ul.collection_nav li {
 .radio {
     padding-left:20px;
 }
+
+#fade {
+    display: none;
+    position:absolute;
+    top: 0%;
+    left: 0%;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/837544010/issues/source?bid=2767031&fileBranchId=2795765#l219)